pages/events/index.tsx

```diff
@@ -46,15 +46,15 @@ const DateTimeCell: FC<{ date: number }> = ({ date }) => {
           color: "common.black",
         }}
       >
-        {DateTime.fromMillis(date).toLocaleString(DateTime.DATE_SHORT)}
+        {DateTime.fromMillis(date).toLocaleString(DateTime.TIME_SIMPLE)}
       </TextCell>
       <TextCell
         sx={{
           fontWeight: 500,
           color: "header.light",
         }}
       >
-        {DateTime.fromMillis(date).toLocaleString(DateTime.TIME_SIMPLE)}
+        {DateTime.fromMillis(date).toLocaleString(DateTime.DATE_SHORT)}
```

Comment:
Maybe we could change this to just show the day of the week instead of the date as a whole.

Reply:
Naw